src/pykeen/models/unimodal/rgcn/model.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of the R-GCN model."""
+
+import logging
+from os import path
+from typing import Any, Mapping, Optional
+
+import torch
+from class_resolver import Hint, Resolver
+from torch import nn
+
+from .decompositions import Decomposition, decomposition_resolver
+from .weightings import EdgeWeighting, edge_weight_resolver
+from ...nbase import ERModel, EmbeddingSpecificationHint
+from ....nn import EmbeddingSpecification, RepresentationModule
+from ....nn.modules import Interaction, interaction_resolver
+from ....triples import TriplesFactory
+from ....typing import Initializer, RelationRepresentation
+
+__all__ = [
+    "RGCN",
+]
+
+logger = logging.getLogger(name=path.basename(__file__))
+
+
+class Bias(nn.Module):
```

Comment:
this should also go in pykeen.nn somewhere since it's not specific for RGCN

Reply:
done in 7c9d7af077f4492d9aa27a65551573c66bb852b2